app/controllers/concerns/foreman_teamdynamix/hosts_controller_extensions.rb

```diff
@@ -0,0 +1,11 @@
+module ForemanTeamdynamix
```

Comment:
Lets leave out all the example files in app until they have real and useful code in them.